psalm.xml.dist

```diff
@@ -59,14 +54,13 @@
                 <referencedClass name="Doctrine\Common\Cache\WinCacheCache"/>
                 <referencedClass name="Doctrine\Common\Cache\ZendDataCache"/>
                 <referencedClass name="Doctrine\Common\Annotations\CachedReader"/>
+                <referencedClass name="Doctrine\ORM\Mapping\Driver\SimplifiedYamlDriver"/>
             </errorLevel>
         </DeprecatedClass>
-        <DeprecatedMethod>
+        <PropertyNotSetInConstructor>
```

Comment:
Can be removed with #61 since `psalm/phpunit-plugin` treats `setUp` method the same way as `__construct`.